plugins/pulp_ostree/plugins/distributors/configuration.py

```diff
@@ -83,3 +96,42 @@ def get_repo_relative_path(repo, config):
     else:
         path = repo.repo_id
     return path
+
+
+def _check_for_relative_path_conflicts(repo_id, relative_path):
+    """
+    Check that a relative path does not conflict with existing distributors' relative paths.
+
+    :param repo_id: identifier of repository associated with the path to check
+    :type  repo_id: basestring
+    :param relative_path: relative path of the repository
+    :type  relative_path: basestring
+    :return error_messages: a list of validation errors
+    :rtype: list
+    """
```

Comment:
I agree. I decided to leave the `_url` in the variable names because this is nearly identical to the RPM method of the same name. My hope is that by using the same variable names, it will be very very easy to combine these into a single function (in platform) in a future X release of pulp after renaming the `relative_url` to `relative_path` in `pulp_rpm`.